src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
Yep, so we are making RepositoryContext IDisposable. Then it gets wrapped into AppContext, which needs to be diposable. To create a consistent behavior, why not making all context classes disposable?